config/locales/en.yml

```diff
@@ -60,9 +60,9 @@ en:
       locked: Locked
   deploy_spec:
     hint:
-      deploy: Impossible to detect how to deploy this application. Please define `deploy.override` in your shipit.yml
-      rollback: Impossible to detect how to rollback this application. Please define `rollback.override` in your shipit.yml
-      fetch: Impossible to detect how to fetch the deployed revision for this application. Please define `fetch` in your shipit.yml
+      deploy: Impossible to detect how to deploy this application. Please define `deploy.override` in your shipit.$SHIPIT_ENVIRONMENT.yml
```

Comment:
Hum, actually, the strict environment thing is optional, so this may be confusing for other users.

Also this `$SHIPIT_ENVIRONMENT` isn't interpolated, so not sure that really helps much.